pyproject.toml

```
@@ -1,16 +1,17 @@
-[tool.poetry]
+[project]
 name = "napalm-unifi"
-version = "0.3.1"
+version = "0.3.2"
 description = ""
-authors = ["Andrew Bates <andrew.bates@networktocode.com>"]
+authors = [
+	{name = "Andrew Bates", email = "andrew.bates@networktocode.com"}
+]
 license = "Apache 2.0"
 readme = "README.md"
 packages = [{include = "napalm_unifi"}]
-
-[tool.poetry.dependencies]
-python = "^3.9"
-napalm = "^4.1.0"
-
+requires-python = "^3.9"
+dependencies = [
+	"napalm >=5.0"
+]
```

Why not just change it to this?

```suggestion
[tool.poetry.dependencies]
python = "^3.9"
napalm = "^5.0"```

That's the change I made to my local copy and re-ran `poetry lock` and everything seems fine. In fact that's the only thing I did and everything appears to work as I would expect.

---

This looks fine! I'm new to poetry and seems that I added the depencies using the old way.